### ISSUE TITLE
`codeView.update` event doesn't fire

### ISSUE THREAD

**Issue by sirlancelot** [froala/wysiwyg-editor#3647]
##### Expected behavior.
Event callback should fire when switching to html view.

##### Actual behavior.
Event does not fire when switching to html view.

##### Steps to reproduce the problem.
[`codeView.update` documentation](https://www.froala.com/wysiwyg-editor/docs/events#codeView.update)

```js
import FroalaEditor from "froala-editor"
import "froala-editor/js/plugins/code_view.min.js"

const editor = new FroalaEditor(node, {
  events: {
    "codeView.update": () => {
      debugger
    }
  }
})
```

CodeMirror is not used.

##### Editor version.
froala-editor@3.0.5 (via NPM)

##### OS.
All OSes

##### Browser.
All Browsers

**Comment by petrinecp** [comment]
Just an observation -> the event fires when you switch to fullscreen mode.

**Comment by sirlancelot** [comment]
I just recently figured that out as well. It seems that event name is used to indicate the desired side-effect which is an anti-pattern. Event names should indicate what is happening, not what their intended effect should be.

**Comment by jmiserez** [comment]
Had the same issue. Workaround: Listen to the 'commands.after' event, it will fire when clicking the code view button (cmd === 'html') and the fullscreen button (cmd === 'fullscreen').

**Comment by kapil2704** [comment]
I have updated code to fire event when you switch from code view to html view. It will be available in 3.0.7 release
